#590: Fix Asciidoc files in documentation causing errors (#670)
devonfw/IDEasy: documentation/LICENSE.adoc

= License
:toc:
toc::[]

The product `IDEasy` is licensed under the following terms.

Binaries of this product have been made available to you by https://devonfw.com[devonfw] under the https://github.com/devonfw/ide/blob/master/LICENSE[Apache License 2.0].

The documentation of this product is licensed under the terms of the http://creativecommons.org/licenses/by-nd/4.0/[Creative Commons License (Attribution-No Derivatives 4.0 International)].

> The source code for this product is available under free and open-source licenses.

Reviewer: ```suggestion
The source code of this product is available under free and open-source licenses.
```

> All the source code to this product is available under licenses which are both free and open source.

Reviewer 2: ```suggestion
The source code for this product is available under free and open-source licenses.
```
